tests/tmp0qrt90_l/butler.yaml

```diff
@@ -0,0 +1,14 @@
+datastore:
```

Comment:
Uh... I think you accidentally committed this temp file and the sqlite db instead of deleting them.

But while you're at it, you may as well add `tests/tmp*` to the root `.gitignore`.

Reply:
Hmmm, I didn't add those files. But I do remember one time that I was running a test and my connection to USDF broke. Maybe that left some straggler files, so adding that directory to `.gitignore` sounds like a good idea.